examples/credit-risk-end-to-end/02_Deploying_the_Feature_Store.ipynb

```diff
@@ -0,0 +1,913 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "21b354ab-ec22-476d-8fd9-6ffe0f3fbacb",
+   "metadata": {},
+   "source": [
+    "Let's use the SDK feature store object (that we set up above) to \"materialize\" data; this is Feast lingo for moving/updating data from the offline store to the online store."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "id": "ff6146df-03a7-4ac2-a665-ee5f440c3605",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING:root:_list_feature_views will make breaking changes. Please use _list_batch_feature_views instead. _list_feature_views will behave like _list_all_feature_views in the future.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Materializing \u001b[1m\u001b[32m3\u001b[0m feature views from \u001b[1m\u001b[32m2023-09-24 12:00:00-06:00\u001b[0m to \u001b[1m\u001b[32m2024-01-07 12:00:00-07:00\u001b[0m into the \u001b[1m\u001b[32msqlite\u001b[0m online store.\n",
+      "\n",
+      "\u001b[1m\u001b[32mdata_b\u001b[0m:\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "  0%|                                                                       | 0/800 [00:00<?, ?it/s]WARNING:root:Cannot use sqlite_vec for vector search\n",
+      "100%|██████████████████████████████████████████████████████████| 800/800 [00:00<00:00, 32269.77it/s]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[1m\u001b[32mdata_a\u001b[0m:\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████████████████████████████████████████████████████| 800/800 [00:00<00:00, 42909.03it/s]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[1m\u001b[32mdata_test\u001b[0m:\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████████████████████████████████████████████████████| 200/200 [00:00<00:00, 19657.42it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Materialize\n",
+    "# Recall that we mocked the outcome data to have timestamps from \n",
+    "# 'Tue Sep 24 12:00:00 2023'out to \"Wed Oct  9 12:00:00 2023\"\n",
+    "# The loan outcome timestamps were then lagged by 30-90 days (which is Jan 7 12:00:00 2024)\n",
+    "res = store.materialize(\n",
+    "    start_date=dt.datetime(2023,9,24,12,0,0),\n",
+    "    end_date=dt.datetime(2024,1,7,12,0,0)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c053fd7b-336d-4286-a3aa-80efee43d2d6",
+   "metadata": {},
+   "source": [
+    "Now, we can query the SQLite database again and see that we get data in the response! Unfortunately, the data is stored in binary strings."
```

Comment:
Should we show this step? Feast internally abstracts this based on the underlying store. Also IMO may not be helpful when the data is binary as a feast user.

Reply:
I'd like to keep it because it provides a bookend to the first db query (input 6). I did add a sentence explaining why the data is in this format, and I think this helps the user visualize some of the optimization that is happening with online stores.